Subsurface/Node/Common/Peripherals/inc/can_interface.h

```diff
@@ -0,0 +1,67 @@
+#ifndef NODE_CAN_INTERFACE_H

```

Comment:
I run git pull origin master. And here are some stuff that is here but not in master. Not sure if it's needed. Do take a look at this.